### PR TITLE
chore: add PR template

### PR DESCRIPTION
**Pull request by dickie81**
Adds PR template to repo.

### CODE REVIEW

**Review by Budaa — DISMISSED** [review]
Ahh, so that's how you add template!

**Comment by dickie81** [comment]
I hope so! Find out if it works if it works!

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+## Description
+
+## Known Issues:
+
+### Checklist for this pull request
+
+* [ ] PR is scoped to one task
+* [ ] Tests included
+* One of:
+* [ ] The PR is not making any UI change
```
Comment (francois-codes) [inline .github/pull_request_template.md]: I think you need to indent those
https://github.com/applicaster/zapp/blob/master/.github/PULL_REQUEST_TEMPLATE.md

**Review by francois-codes — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (dickie81) [reply .github/pull_request_template.md]: Can you give me access to that repo?

Comment (francois-codes) [reply .github/pull_request_template.md]: you are now part of applicaster team, meaning you have read access on everything (plus write access on RN repos as member of team React-Native)

**Comment by dickie81** [comment]
Updated. @f-roland thanks for adding me to the team.

**Review by francois-codes — APPROVED** [review]

**Pull request merged**